### ISSUE TITLE
Auto sync interval vs. needless traffic at servers

### ISSUE THREAD

**Issue by mMuck** [NeoApplications/Neo-Store#485]
I agree with issue #280 but the current limit of 12 h (43200 s) should be increased IMO to limit the load and traffic to the servers as these are driven by donations.

My personal setting would be one week as I don't want to be disturbed by notifications every some hours. Could be that dev people love updating apps more often but average users will interpret these notifications as spam.

Perhaps the unit could be changed to hours or a list type like 4 h, 8 h, 12 h, 1 d, 3 d, 5 d, 7 d, 14 d. Current settings could easily be translated into this (43200 -> 8 h; <43200 -> 4 h). Shorter intervals should be forbidden.

In other words: I wouldn't like if f-droid burns my donations due to traffic costs resulting from Neo Store auto-syncs 😉. Thanks

**Comment by machiav3lli** [comment]
Actually you've misread, as 43200 are minutes not seconds (720 hours). And just to make a point clear, longer intervals can be unreliable as the Android system is responsable for resources management and may decide to remove these jobs…

That said I changed the unit to hours with range of 1-720.

> Shorter intervals should be forbidden.

Users can (and should be able to) sync manually, which defeats the purpose of this.

**Comment by mMuck** [comment]
I'm sorry, i confused minutes and seconds. So 43200 were already 30 days which is fine. The hour unit is much handier. Thanks.

> Users can (and should be able to) sync manually, which defeats the purpose of this.

I'm fine with this. I meant the interval for auto sync not deactivating manual syncs. An interval < 60 min was possible. Now it is 1 h minimum. Thanks.
